src/compute/flexbox.rs

```diff
@@ -1800,15 +1828,24 @@ fn calculate_layout_line(
 
 /// Do a final layout pass and collect the resulting layouts.
 #[inline]
-fn final_layout_pass(tree: &mut impl PartialLayoutTree, flex_lines: &mut [FlexLine], constants: &AlgoConstants) {
+fn final_layout_pass(
+    tree: &mut impl PartialLayoutTree,
+    flex_lines: &mut [FlexLine],
+    constants: &AlgoConstants,
+) -> Size<f32> {
     let mut total_offset_cross = constants.content_box_inset.cross_start(constants.dir);
 
+    #[cfg_attr(not(feature = "content_size"), allow(unused_mut))]
```

Comment:
Why is the unused mut fine here?

Reply:
Well, unused `mut` is harmless in general. Except that it's confusing for readers of the code. In this case the variable is mutable when the feature is enabled and immutable otherwise. My judgement here was that it was more readable to do this than conditionally define the variable depending on the feature flag which would look like:

```rust 
#[cfg(feature = "content_size")]
let mut content_size = Size::ZERO;
#[cfg(not(feature = "content_size"))]
let content_size = Size::ZERO;
```